src/main/kotlin/com/sourcegraph/utils/CodyFormatter.kt

```diff
@@ -34,6 +34,7 @@ class CodyFormatter {
                 .createFileFromText("TEMP", file.fileType, appendedString)
 
         val codeStyleManager = CodeStyleManager.getInstance(project)
+        // FIXME this will fail if cursor < range.startOffset + completionText.length
```

Comment:
Is that going to be fixed as part of this PR?

Reply:
I have fixed my code that was passing wrong parameters, but IMO there is a problem with the interface of `CodyFormatter.formatStringBasedOnDocument`. I would rather leave it for another time.